components/clp-py-utils/clp_py_utils/profiling_utils.py

```
@@ -0,0 +1,256 @@
+
+                try:
+                    result = await func(*args, **kwargs)
+                    return result
+                finally:
+                    profiler.stop()
+                    _save_profile(profiler, name, job_id, task_id)
+
+            return async_wrapper  # type: ignore
+
+        @functools.wraps(func)
+        def sync_wrapper(*args, **kwargs):
```

Ruff (0.14.1)

Missing return type annotation for private function `sync_wrapper`

(ANN202)